probe_benchmark/build_df_scaling_experiments.py

```diff
@@ -0,0 +1,116 @@
+import os
+import pandas as pd
+import json
+
+if __name__ == '__main__':
+
+    compute_df = pd.read_csv('probe_benchmark/clip_table_2.csv')
+    #mdf = pd.read_csv("https://gist.githubusercontent.com/mehdidc/58dee67cecd5431a80ee3a2346c9c165/raw/45288ebccaacc34a97f580f8bf16fb3274927f2c/gistfile1.txt")
+    mdf = pd.read_csv("probe_benchmark/openclip_results.csv")
+    info = []
+    #import pdb; pdb.set_trace()
+    models = ['ViT-B-32-quickgelu,laion400m_e32',
+              'ViT-B-32,openai',
+              'ViT-B-32,laion2b_s34b_b79k',
+              'ViT-B-16,laion400m_e32',
+              'ViT-B-16-plus-240,laion400m_e32',
+              'ViT-B-16,openai',
+              #'ViT-L-14-336,openai',
+              'ViT-L-14,openai',
+              'ViT-B-32,laion2b_e16',
+              'ViT-L-14,laion400m_e32',
+              'ViT-L-14,laion2b_s32b_b82k',
+              'ViT-H-14,laion2b_s32b_b79k',
+              'ViT-g-14,laion2b_s12b_b42k',
+            ]
+    alt_models = ['B/32 400M',
+            'B/32 CLIP WIT',
+            'B/32 2B',
+            'B/16 400M',
+            'B/16+ 400M',
+            'B/16 CLIP WIT',
+            #'ViT-L-14-336,openai',
+            'L/14 CLIP WIT',
+            'B/32 2B',
+            'L/14 400M',
+            'L/14 2B',
+            'H/14 2B',
+            'g/14 2B',
+          ]
+
+    datasets = ['imagenet1k-unverified', 'cifar100']
+    datasets = datasets + [
+      'vtab/caltech101',
+      'vtab/cifar10',
+      'vtab/cifar100',
+      'vtab/clevr_count_all',
+      'vtab/clevr_closest_object_distance',
+      'vtab/diabetic_retinopathy',
+      'vtab/dmlab',
+      'vtab/dsprites_label_orientation',
+      'vtab/dsprites_label_x_position',
+      'vtab/dtd',
+      'vtab/eurosat',
+      'vtab/kitti_closest_vehicle_distance',
+      'vtab/flowers',
+      'vtab/pets',
+      'vtab/pcam',
+      'vtab/resisc45',
+      'vtab/smallnorb_label_azimuth',
+      'vtab/smallnorb_label_elevation',
+      'vtab_svhn',
```

Comment:
@mitchellnw Shouldn't this be `vtab/svhn` or there was a specific issue there?

Reply:
resolved